pytest_mpl/plugin.py

```diff
@@ -131,7 +144,7 @@ def item_function_wrapper(*args, **kwargs):
 
             baseline_remote = baseline_dir.startswith('http')
 
-            with plt.style.context(style):
+            with plt.style.context(style), switch_backend(backend):
```

Comment:
Only on the 1.4 series--so I guess the question is when does pytest-mpl drop 1.4?